Services/Models/DeviceTwin.cs

```diff
@@ -34,35 +34,17 @@ public DeviceTwin(Twin twin)
             }
         }
 
-        /*
-        JValue:  string, integer, float, boolean
-        JArray:  list, array
-        JObject: dictionary, object
-
-        JValue:     JToken, IEquatable<JValue>, IFormattable, IComparable, IComparable<JValue>, IConvertible
-        JArray:     JContainer, IList<JToken>, ICollection<JToken>, IEnumerable<JToken>, IEnumerable
-        JObject:    JContainer, IDictionary<string, JToken>, ICollection<KeyValuePair<string, JToken>>, IEnumerable<KeyValuePair<string, JToken>>, IEnumerable, INotifyPropertyChanged, ICustomTypeDescriptor, INotifyPropertyChanging
-        JContainer: JToken, IList<JToken>, ICollection<JToken>, IEnumerable<JToken>, IEnumerable, ITypedList, IBindingList, IList, ICollection, INotifyCollectionChanged
-        JToken:     IJEnumerable<JToken>, IEnumerable<JToken>, IEnumerable, IJsonLineInfo, ICloneable, IDynamicMetaObjectProvider
-        */
         private static Dictionary<string, JToken> TwinCollectionToDictionary(TwinCollection x)
         {
             var result = new Dictionary<string, JToken>();
 
             if (x == null) return result;
 
-            foreach (KeyValuePair<string, object> twin in x)
+            foreach (KeyValuePair<string, JToken> twin in x)
```

Comment:
As mentioned earlier, it's ok refactoring this class, however we need to test all the scenarios if we do so. We know that the current mapping function works. To avoid regressions we need tests.